gitpod-io/gitpod
# Implement user account verification with LinkedIn during onboarding

## Description


Implement user account verification with LinkedIn during onboarding.

## Related Issue(s)

Fixes #

## How to test


1. Try onboading flow
2. Verify your account by connecting to LinkedIn

## Release Notes

```release-note
Implement user account verification with LinkedIn during onboarding
```

## Documentation


## Build Options:

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
